### ISSUE TITLE
The location template tag should probably return None

### ISSUE THREAD

**Issue by danielquinn** [Bouke/django-user-sessions#67]
Currently, if `location()` fails to find anything, it returns

```html
<i>unknown</i>
```

Which is problematic on two fronts:

1. The `<i>` tag has been deprecated for a while in favour of `<em>`.
2. It limits the user's ability to control the output.  Most filters return `None` if nothing is available because it allows you to do stuff like this:

    ```html
    {{ session.ip|locaiton|default_if_none:'Nothing found' }}
    ```

For that matter, the `device` template tag has similar behaviour.

I can offer a pull request if you agree with this assessment.  I just didn't want to write one if you wouldn't want to use it.

**Comment by Bouke** [comment]
Seems like a valid change request, I'll welcome the PR.

**Comment by Bouke** [comment]
Fixed in adbcd552a2de465e8da9c49ccfe51d8bcadf83ea.
